packages/moonstone/Scrollable/Scrollable.js

```diff
@@ -595,11 +595,15 @@ class ScrollableBase extends Component {
 			let direction = null;
 
 			if (isPageUp(keyCode) || isPageDown(keyCode)) {
-				if (this.isContent(target) && (this.props.direction === 'vertical' || this.props.direction === 'both')) {
-					direction = isPageUp(keyCode) ? 'up' : 'down';
-					this.scrollByPage(direction);
-					if (overscrollEffectOn.pageKey) { /* if the spotlight focus will not move */
-						this.checkAndApplyOverscrollEffectByDirection(direction);
+				if (this.props.direction === 'vertical' || this.props.direction === 'both') {
+					if (this.isContent(target)) {
+						direction = isPageUp(keyCode) ? 'up' : 'down';
+						this.scrollByPage(direction);
+						if (overscrollEffectOn.pageKey) { /* if the spotlight focus will not move */
+							this.checkAndApplyOverscrollEffectByDirection(direction);
+						}
+					} else if (this.props.overscrollEffectOn.pageKey) {
```

Comment:
If we will not check whether keypress happens on scrollbar or not, this clause works for all inputs when `this.isContent(target)` is false. If possible, let's change `isContent` into a proper condition to cover all area.